numba/cuda/tests/cudadrv/test_linker.py

```diff
@@ -158,6 +159,14 @@ def kernel(x):
         # Check the filename is reported correctly
         self.assertIn('in the compilation of "error.cu"', msg)
 
+    @skip_with_cuda_python
+    def test_linking_cu_ctypes_unsupported(self):
+        msg = 'not supported with the ctypes binding'
```

Comment:
Perhaps:
```suggestion
        msg = ('Linking CUDA source files is not supported with the ctypes '
               'binding')
```
just to make absolutely sure it's that specific exception from linking and not a different but similar one?